<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindowWidget</class>
 <widget class="QDialog" name="ProfileWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профиль</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="contractsTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="namePushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passwordPushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phonePushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="saleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sale</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
